src/libtls/tls_hkdf.h

```diff
@@ -137,6 +137,22 @@ struct tls_hkdf_t {
 	bool (*resume)(tls_hkdf_t *this, chunk_t messages, chunk_t nonce,
 				   chunk_t *psk);
 
+	/**
+	 * Generate a PSK binder.
+	 *
+	 * @note The transcript hash is build of the partial ClientHello message up
+	 * to and including the PreSharedKey extension identities field as of
+	 * RFC 8446, section 4.2.11.2. The length of the psk_binder is either
+	 * 32 byte for SHA256 or 48 bytes for SHA384. PSK_binder is defined as a
+	 * list of hashes, each hash has a one byte length and all hashes together a
+	 * two byte length specifier in front.
```

Comment:
I don't think it's necessary to describe the actual encoding of the binders (or their length, see the other comment). Important is that the binders are encoded (e.g. as empty chunks of the right length) in the extension and message, just not hashed.
```suggestion
	 * @note The transcript hash is built of the partial ClientHello message up
	 * to and including the PreSharedKey extension's identities field, excluding
	 * the actual binders (their length is included in that of the extension(s)
	 * and message, though), as per RFC 8446, section 4.2.11.2.
```